include/openenclave/bits/evidence.h

```diff
@@ -62,29 +56,88 @@ typedef struct _oe_claim
     size_t value_size;
 } oe_claim_t;
 
-/**
+/*
  * Claims that are known to OE that every attestation plugin should output.
  */
+
+/**
+ * Claims version.
+ */
 #define OE_CLAIM_ID_VERSION "id_version"
+
+/**
+ * Security version of the enclave. (SVN for SGX).
+ */
 #define OE_CLAIM_SECURITY_VERSION "security_version"
+
+/**
+ * Attributes flags for the evidence.
+ */
 #define OE_CLAIM_ATTRIBUTES "attributes"
+
+/**
+ * The unique ID for the enclave (MRENCLAVE for SGX).
+ */
 #define OE_CLAIM_UNIQUE_ID "unique_id"
+
+/**
+ * The signer ID for the enclave (MRSIGNER for SGX).
+ */
 #define OE_CLAIM_SIGNER_ID "signer_id"
+
+/**
+ * The product ID for the enclave (ISVPRODID for SGX).
+ */
 #define OE_CLAIM_PRODUCT_ID "product_id"
+
+/**
+ * The format id of the evidence.
+ */
 #define OE_CLAIM_FORMAT_UUID "format_uuid"
```

Comment:
Should "format_uuid" be changed to "format_id"?

Reply:
This has been published. Will be a breaking change if we change. I'd leave it as is.

Reply:
Was it published as "experimental"?  If so then breaking changes are ok.